docs/calculations.mdx

```diff
@@ -313,32 +313,32 @@ creative_data_transfer_bytes =
       ad_format.video_player.size_bytes +
       ad_format.other_asset_bytes
 
-creative_data_transfer_usage_emissions_gco2pm =
+creative_data_transfer_usage_emissions_gco2 =
       creative_data_transfer_bytes / 1024 x
       usage_kwh_per_gb / 1000000 x
       lookup_carbon_intensity_gco2e_per_kwh(country, region, utc_datetime)
 
-creative_data_transfer_embodied_emissions_gco2pm =
+creative_data_transfer_embodied_emissions_gco2 =
       creative_data_transfer_bytes / 1024 x
       embodied_emissions_gco2e_per_kb
 ```
 
 If data transfer is provided in the delivery row, use it:
 
 ```
-image_bytes = creative_image_data_transfer_bytes
-video_bytes = creative_video_data_transfer_bytes + creative_video_vast_bytes
-audio_bytes = creative_audio_data_transfer_bytes
+image_bytes = creative_image_data_transfer_bytes * impressions
```

Comment:
Is this correct or are we expecting them to give us the full image data transfer for all impressions or just a single impression

Reply:
I was thinking you'd pass in bytes and impressions. If it's per imp we should say "average bytes"